subsys/bluetooth/host/iso.c

```diff
@@ -440,6 +442,28 @@ static void bt_iso_chan_disconnected(struct bt_iso_chan *chan, uint8_t reason)
 			 */
 			bt_iso_remove_data_path(chan->iso);
 		}
+
+#if defined(CONFIG_BT_ISO_UNICAST)
+		bool fully_disconnected;
+		struct bt_iso_cig *cig;
+
+		/* Update CIG state */
+		cig = get_cig(chan);
+		__ASSERT(cig != NULL, "CIG was NULL");
+
+		fully_disconnected = false;
+		SYS_SLIST_FOR_EACH_CONTAINER(&cig->cis_channels, chan, node) {
+			if (chan->state == BT_ISO_CONNECTED ||
+			    chan->state == BT_ISO_CONNECT) {
+				fully_disconnected = false;
+				break;
+			}
+		}
+
+		if (fully_disconnected) {
```

Comment:
Since the `if` clause checks `BT_ISO_CONNECTED` or `BT_ISO_CONNECT`, use `is_chan_connected` as the variable. This will ensure the boolean is assigned false, and `!is_chan_connected` be used here.